Added the app design figma file

ara2105 opened the pull request:
Fixes #18 

Here is the Zplit app design - https://www.figma.com/design/MzAfV4ex1zItWjsOu4Tk5I/Zplit---Aradhya?t=SvChlLcttDv30Swo-1

Created a new folder and added the .fig file for the same.

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->

## Summary by CodeRabbit

* **Documentation**
  * Added documentation for extension discovery cache configuration and behavior.

* **Chores**
  * Added extension discovery manifest file for development tooling support.

<sub>✏️ Tip: You can customize this high-level summary in your review settings.</sub>

<!-- end of auto-generated comment: release notes by coderabbit.ai -->

ara2105 committed:
Added the app design figma file

M4dhav commented:
Please rename the file to zplit-design, and for future reference, not that all PRs must be raised to dev instead of main

ara2105 commented:
Okay @M4dhav Understood, will do the asked.

ara2105 committed:
modified file name

M4dhav commented:
please rename it to exactly `zplit-design`, all lowercase and with a dash seperator instead of space. Also, please do not use force pushes

M4dhav commented:
> please rename it to exactly `zplit-design`, all lowercase and with a dash seperator instead of space. Also, please do not use force pushes

@ara2105

ara2105 commented:
Oh okay sure

M4dhav commented on design/Zplit Design.fig:
Please remove the old file from the PR

ara2105 committed:
Remove old design file and clean up figma asset

ara2105 commented:
Hey @M4dhav  , this is resolved now. I’ve removed the old design file and cleaned up the PR.
Sorry about the force push, I had to do it to fix the commit history. Thanks for the review!

M4dhav commented:
Thank you for the contribution
